editor/inserter/menu.js

```diff
@@ -68,10 +68,15 @@ class InserterMenu extends wp.element.Component {
 	}
 
 	hoverBlock() {
+		const { lastMultiSelectedBlock, selectedBlock } = this.props;
+		let insertionPoint = null;
```

Comment:
In a subsequent pull request, we should think about consolidating or simplifying some of this shared logic between the menu and the root component. I still think it's a bit bizarre how we assign insertion point in the hover intent, but I think I recall the reasoning as to show the indicator in the post.